zuul.d/edpm.yaml

```diff
@@ -1,11 +1,22 @@
 ---
-# Base job for openstack crc based job
+# Base job for crc based job
 - job:
-    name: cifmw-base-crc-openstack
+    name: cifmw-base-crc
     nodeset: centos-9-crc-3xl
     timeout: 10800
     abstract: true
     parent: base-simple-crc
+    vars:
+      crc_parameters: "--memory 21000 --disk-size 120 --cpus 8"
+      pre_pull_images:
+        - registry.redhat.io/rhosp-rhel9/openstack-rabbitmq:17.0
+
+# Base job for openstack based job containing ci-framework bits
+- job:
+    name: cifmw-base-openstack
```

Comment:
I think it's worth keeping the job name as `cifmw-base-crc-openstack` since its parent is setting up a crc environmet. I see that you are keeping the old name for compatibility, but as personal opinion, looks like the old name still represents it better.

Reply:
Same thoughts - seems to be corrected, thanks @raukadah !